src/ansys/aedt/core/modules/solve_setup.py

```diff
@@ -838,6 +839,18 @@ def add_mesh_link(
         ----------
 
         >>> oModule.EditSetup
+
+        Examples
+        --------
+        This example shows how to import the mesh from a source design solution.
+        The target design is duplicated from the source design, and then the mesh
+        is imported.
+
+        >>> from ansys.aedt.core import Maxwell3d
+        >>> m3d = Maxwell3d(design="source_design")
+        >>> m3d.create_setup(name="setup1")
+        >>> m3d.duplicate_design(name="source_design", save_after_duplicate=True)
+        >>> m3d.setups[0].add_mesh_link(design="source_design", solution="setup1 : LastAdaptive")
```

Comment:
@anur7 I'd put here an explanation that the active design in now the duplicated one and you are assigning the mesh link to its setup.
Also please check that _self.aedtapp.nominal_adaptive_ gives the same info in solution. This should give you back "setup1 : LastAdaptive".
Please add the release_desktop at the end.

Reply:
yes, @gmalinve self.aedtapp.nominal_adaptive gives you "setup1 : LastAdaptive". Do you want me to subtitute the str with that comand?

Reply:
@anur7 yes, because this is what "solution" needs.

Reply:
@gmalinve I added your suggestion, please check